docs/docs/writing-hello-world.md

```diff
@@ -77,30 +79,34 @@ each of the different build types.
 
 * Simulation builds use the default dummy key.
 * Debug builds use a private key stored in a file.
-* Release builds use a private key managed by some external signing process.
+* Release builds use a private key managed by an external signing process.
+
+2. Copy the `signing` directory in the `hello world`
```

Comment:
OMG, we should not be telling users to copy a sample private key!!

The instructions here will need to updated to tell the user how to generate their own signing key.

Reply:
Removed it and added an instruction to generate a signing key along with a link to more details.